### ISSUE TITLE
v3.0.2 doesn't have dist folder

### ISSUE THREAD

**Issue by Kikobeats** [gajus/http-terminator#31]
I'm getting this error 

```
Error: Cannot find module 'node_modules/http-terminator/dist/src/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:353:19)
    at Function.Module._findPath (node:internal/modules/cjs/loader:566:18)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:94:18)
    at Object.<anonymous> (/Users/kikobeats/Projects/microlink/api/node_modules/lightship/dist/src/factories/createLightship.js:11:27)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
```

That the error is right, the folder `dist` was not published with the release

<img width="804" alt="CleanShot 2021-09-18 at 11 45 55@2x" src="https://user-images.githubusercontent.com/2096101/133884636-454356aa-b756-47e2-a6d3-1c8de78efa6c.png">

**Comment by schimini** [comment]
Also, package.json main is configured wrong, it points to dist/src/....js but when I ran the build command the src folder wasn't created inside dist

**Comment by schimini** [comment]
I suppose that 3.0.3 fixes this.

**Comment by Kikobeats** [comment]
fixed at `v3.0.3`
